verifier/verifier.go

```diff
@@ -64,15 +64,15 @@ func New(trustPolicy *trustpolicy.Document, trustStore truststore.X509TrustStore
 }
 
 // SkipVerify validates whether the verification level is skip.
-func (v *verifier) SkipVerify(ctx context.Context, artifactRef string) (bool, *trustpolicy.VerificationLevel, error) {
+func (v *verifier) SkipVerify(ctx context.Context, opts notation.VerifierVerifyOptions) (bool, *trustpolicy.VerificationLevel, error) {
 	logger := log.GetLogger(ctx)
 
-	logger.Debugf("Check verification level against artifact %v", artifactRef)
-	trustPolicy, err := v.trustPolicyDoc.GetApplicableTrustPolicy(artifactRef)
+	logger.Debugf("Check verification level against artifact %v", opts.ArtifactReference)
+	trustPolicy, err := v.trustPolicyDoc.GetApplicableTrustPolicy(opts.ArtifactReference)
 	if err != nil {
 		return false, nil, notation.ErrorNoApplicableTrustPolicy{Msg: err.Error()}
 	}
-	logger.Debugf("Trust policy configuration: %+v", trustPolicy)
+	logger.Infof("Trust policy configuration: %+v", trustPolicy)
```

Comment:
Shouldn't this be debug else it will get printed on console?

Reply:
Resolving as we had an offline discussion on it.